ODIN_II/SRC/parse_making_ast.cpp

```diff
@@ -1783,15 +1818,15 @@ void graphVizOutputAst(std::string path, ast_node_t* top) {
     static int file_num = 0;
 
     /* open the file */
-    odin_sprintf(path_and_file, "%s/%s_ast.dot", path.c_str(), top->children[0]->types.identifier);
+    odin_sprintf(path_and_file, "%s/%s_ast.dot", path.c_str(), top->identifier_node->types.identifier);
     fp = fopen(path_and_file, "w");
     file_num++;
 
     /* open graph */
     fprintf(fp, "digraph G {\t\nranksep=.25;\n");
```

Comment:
while you are changing it, would you mind modifying this to 
```
fprintf(fp, "digraph G {\t\nrankdir=LR;\t\nranksep=.25;\n");
```
this will print the graph left to right, making it much more readable

Reply:
The line under open graph right?

Reply:
Yep